src/corecel/Types.hh

```diff
@@ -79,6 +79,25 @@ enum class UnitSystem
     native = CELERITAS_UNITS,  //!< Compile time selected system
 };
 
+//---------------------------------------------------------------------------//
+//! Distributions returning a single real type
+enum class OnedDistributionType
+{
+    delta,
+    normal,
+    size_
+};
+
+//---------------------------------------------------------------------------//
+//! Distributions returning a length three array of real types
+enum class ThreedDistributionType
+{
+    delta,
+    isotropic,
+    uniform_box,
+    size_
+};
+
```

Comment:
I wonder when we should come up with some rules about the granularity of `Types.hh` files: I know I shouldn't be worried about it, but it kind of bothers me that adding a new 1D distribution will cause the whole codebase to recompile.